src/watertap_contrib/reflo/solar_models/surrogate/flat_plate/data/pyssc.py

```diff
@@ -0,0 +1,534 @@
+import sys, os
```

Comment:
Add header?

Reply:
This is sort of a custom implementation of PySAM as I understand it, not sure we should be putting a header on this one.